macros/generate_daily_time_series_values.sql

```diff
@@ -8,13 +8,13 @@
        SUPPORTS:
             - Postgres
             - Snowflake
-    #}
+    */#}
 
-    {%- if target.type in ['postgres'] -%} 
-        generate_series
-    {%- elif target.type == 'snowflake' -%}
-        dateadd(day, '+' || row_number() over (order by seq4()), TO_TIMESTAMP('{{ start_date }}') - INTERVAL '1 day')
-    {%- else -%}
-        {{ xdb.not_supported_exception('generate_time_series_values') }}
-    {%- endif -%}
+{%- if target.type in ['postgres'] -%} 
+    generate_series
+{%- elif target.type == 'snowflake' -%}
+    dateadd(day, '+' || row_number() over (order by seq4()), TO_TIMESTAMP('{{ start_date }}') - INTERVAL '1 day')
```

Comment:
I think `dateadd`, `row_number()`, `over`, and `order by` should all be capitalized.